lib/helpers.js

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as credentialsContext from 'credentials-context';
+import * as credentialsContextV2 from '@digitalbazaar/credentials-v2-context';
+
+export const {constants: {CREDENTIALS_CONTEXT_V1_URL}} = credentialsContext;
+export const {
+  constants: {
+    CONTEXT_URL: CREDENTIALS_CONTEXT_V2_URL
+  }
+} = credentialsContextV2;
+
+const credentialContextUrls = new Set([
+  CREDENTIALS_CONTEXT_V1_URL,
+  CREDENTIALS_CONTEXT_V2_URL
+]);
+
+/**
+ * The error thrown if the first context is not
+ * a credentials context.
+ */
+export const ContextOrderError = new Error(
+  `"${CREDENTIALS_CONTEXT_V1_URL}" or "${CREDENTIALS_CONTEXT_V2_URL}"` +
+  ' needs to be first in the list of contexts.');
```

Comment:
it could be put into a function as it is a really specific error so yeah I'll put it inside of a function. I believe this started out a constant in an external file imported into `index.js` and then it just made more sense to reduce bloat in `index.js` and move the validation logic here. One small issue though: this is spaced the way it's spaced to avoid hitting max-len 80 and your edit adds another space to the line above which probably doesn't cause a linter issue, but as that can't be verified in a PR suggestion I don't think it's a good idea to take the suggestion.